common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala

```diff
@@ -5,90 +5,35 @@ object UsageRightsConfig {
   // Note: we filter exclusively on matching source, rather than combining credit=Getty and source=X
   // this is assumed to be good enough as it's unlikely other provider will use the same source.
   val payGettySourceList = List(
-    "ASAblanca",
-    "Anadolu",
-    "BBC News & Current Affairs",
-    // TODO: put back in once we can filter on both
-    // supplier/collection, since we have a direct deal with
-    // Barcroft. Also, should it be "Barcroft Media"?
-    // "Barcroft",
-    "Blom UK",
-    "Boston Globe",
-    "British Athletics",
-    "Bundesliga Collection",
-    "Caiaimage",
-    "Carnegie Museum Art",
+    // TODO: Remove once we are on the new UsageRights model
+    // This is not excluded as we have a deal with Barcroft,
+    // and there are only a tiny amount from Getty
+    // "Barcroft Media",
     "Catwalking",
-    "Champions Hockey League",
-    "City-Press",
+    "Contour by Getty Images",
     "Contour Style",
-    "Contour",
-    "Country Music Hall of Fame and Museum",
-    "Cricket Australia",
-    "Denver Post",
     "Edit",
-    "Fox Image Collection",
-    "French Select",
-    "Frontzone Sport",
-    "GC Images",
-    "Gallo Images Editorial",
-    "Gamma Legends",
-    "Gamma Rapho",
-    "German Select",
-    "Getty Images Portrait",
-    "Golden Boy Promotions",
-    "Her Og Nu",
-    "Hero Images",
-    "Hulton Royals Collection",
-    "IAAF World Athletics",
-    "ICC",
-    "Imperial War Museums",
-    "Interact Images",
+    "Fashion Window",
+    "Gamma-Legend",
+    "Gamma-Rapho",
     "International Center of Photography",
-    "Kommersant",
     "Lichfield Archive",
-    "LightRocket",
-    "MLB Platinum",
-    "Mark Leech Sports Photography",
+    "MLB Major League Baseball Platinum",
```

Comment:
Neither match - but assuming as the new ones seem to be correct, this must be correct.